sklearn_bench/dbscan.py

```diff
@@ -19,38 +19,41 @@
 import os
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 import bench
-from sklearn.metrics.cluster import davies_bouldin_score
 
-parser = argparse.ArgumentParser(description='scikit-learn DBSCAN benchmark')
-parser.add_argument('-e', '--eps', '--epsilon', type=float, default=10.,
-                    help='Radius of neighborhood of a point')
-parser.add_argument('-m', '--min-samples', default=5, type=int,
-                    help='The minimum number of samples required in a '
-                    'neighborhood to consider a point a core point')
-params = bench.parse_args(parser)
-
-from sklearn.cluster import DBSCAN
-
-# Load generated data
-X, _, _, _ = bench.load_data(params, add_dtype=True)
-
-# Create our clustering object
-dbscan = DBSCAN(eps=params.eps, n_jobs=params.n_jobs,
-                min_samples=params.min_samples, metric='euclidean',
-                algorithm='auto')
-
-# N.B. algorithm='auto' will select DAAL's brute force method when running
-# daal4py-patched scikit-learn, and probably 'kdtree' when running unpatched
-# scikit-learn.
-
-# Time fit
-time, _ = bench.measure_function_time(dbscan.fit, X, params=params)
-labels = dbscan.labels_
-
-params.n_clusters = len(set(labels)) - (1 if -1 in labels else 0)
-acc = davies_bouldin_score(X, labels)
-
-bench.print_output(library='sklearn', algorithm='dbscan', stages=['training'],
-                   params=params, functions=['DBSCAN'], times=[time], accuracies=[acc],
-                   accuracy_type='davies_bouldin_score', data=[X],
-                   alg_instance=dbscan)
+def main():
+    from sklearn.cluster import DBSCAN
+    from sklearn.metrics.cluster import davies_bouldin_score
+
+    # Load generated data
+    X, _, _, _ = bench.load_data(params, add_dtype=True)
+
+    # Create our clustering object
+    dbscan = DBSCAN(eps=params.eps, n_jobs=params.n_jobs,
+                    min_samples=params.min_samples, metric='euclidean',
+                    algorithm='auto')
+
+    # N.B. algorithm='auto' will select DAAL's brute force method when running
```

Comment:
```suggestion
    # N.B. algorithm='auto' will select oneAPI Data Analytics Library (oneDAL) brute force method when running
```

Reply:
@PetrovKP what about other files that @vlad-nazarov did not touch?

Reply:
I will correct if there are such places yet